packages/cadl-python/test/generated/collections-basic/cadl/testserver/collectionpropertiesbasic/_operations/_operations.py

```diff
@@ -128,23 +116,23 @@ def send_collection_model(  # pylint: disable=inconsistent-return-statements
         :keyword content_type: Body Parameter content-type. Content type parameter for binary body.
          Default value is "application/json".
         :paramtype content_type: str
-        :return: None
+        :return: None. This object is MutableMapping-compatible.
         :rtype: None
         :raises ~azure.core.exceptions.HttpResponseError:
         """
 
     @distributed_trace
     def send_collection_model(  # pylint: disable=inconsistent-return-statements
-        self, input: Union[JSON, IO], **kwargs: Any
+        self, input: Union[_models.InputModel, JSON, IO], **kwargs: Any
     ) -> None:
         """send_collection_model.
 
         :param input: Is either a model type or a IO type. Required.
-        :type input: JSON or IO
+        :type input: ~cadl.testserver.collectionpropertiesbasic.models.InputModel or JSON or IO
         :keyword content_type: Body parameter Content-Type. Known values are: application/json. Default
          value is None.
         :paramtype content_type: str
-        :return: None
+        :return: None. This object is MutableMapping-compatible.
```

Comment:
None is not MutableMapping compatible

Reply:
added logic to add this sentence only when the response is a model